### PR TITLE
Improve feature description

### PR DESCRIPTION
**Pull request by lchrusciel**

### CODE REVIEW

File: features/not_crashing_behat_with_cross_container_extension.feature [hunk 1]
```diff
@@ -1,4 +1,7 @@
-Feature: Not crashing Behat
+Feature: Useage of ServiceContainerExtension together with CrossContainerExtension
```
Comment (pamil) [inline features/not_crashing_behat_with_cross_container_extension.feature]: `Useage` -> `Usage`

File: features/not_crashing_behat_with_cross_container_extension.feature [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+    In order to declare own services and parameters inside Behat and other containers
+    As a Behat User
+    I need to use ServiceContainerExtension and ServiceContainerExtension together
```
Comment (pamil) [inline features/not_crashing_behat_with_cross_container_extension.feature]: `ServiceContainerExtension and ServiceContainerExtension` xD

File: features/not_crashing_behat_with_cross_container_extension.feature [hunk 3]
```diff
@@ -1,4 +1,7 @@
-Feature: Not crashing Behat
+Feature: Useage of ServiceContainerExtension together with CrossContainerExtension
+    In order to declare own services and parameters inside Behat and other containers
```
Comment (pamil) [inline features/not_crashing_behat_with_cross_container_extension.feature]: In order to reference cross-container services and parameters inside Behat container

File: features/not_crashing_behat_with_cross_container_extension.feature [hunk 4]
```diff
@@ -7,7 +10,7 @@ Feature: Not crashing Behat
             extensions:
                 FriendsOfBehat\ServiceContainerExtension: ~
 
-                FriendsOfBehat\CrossContainerExtension: ~
+                FriendsOfBehat\ServiceContainerExtension: ~
```
Comment (pamil) [inline features/not_crashing_behat_with_cross_container_extension.feature]: XDXDXD

**Review by pamil — DISMISSED** [review]

File: features/not_crashing_behat_with_cross_container_extension.feature (hunk 4 above)
Comment (Zales0123) [reply features/not_crashing_behat_with_cross_container_extension.feature]: XDXDXDXD

**Review by pamil — APPROVED** [review]
Very good job!

**Comment by pamil** [comment]
Thank you @lchrusciel for bringing so much value to FriendsOfBehat.

image

**Pull request merged**